modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml

```diff
@@ -344,4 +344,10 @@
             <class name="org.wso2.am.integration.tests.workflow.WorkflowApprovalExecutorTest"/>
         </classes>
     </test>
+
+    <test name="apim-integration-tests-benchmark" preserve-order="true" parallel="false" group-by-instances="true">
+        <classes>
+            <class name="org.wso2.am.integration.tests.benchmarktest.BenchmarkTestCase"/>
```

Comment:
how long does this take to run completely?

Reply:
About 15 minutes, including the tenant iterations.

Reply:
we need to reduce this. let's discuss more at the code review.